src/components/VideosList/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import VideosListElement from '../VideosListElement';
+
+const VideosList = ({ videos, setVideo, setVideos }) => {
+  const updateView = (video) => {
+    setVideo(video);
+    setVideos(videos);
+  };
+
+  return (
+    <div>
+      {videos.map((video) =>
+        video.id.videoId ? (
+          <Link
+            key={video.id.videoId}
+            to={{ pathname: `/${video.id.videoId}`, state: { video } }}
+            onClick={() => updateView(video)}
+          >
+            <VideosListElement key={video.id.videoId} snippet={video.snippet} />
+          </Link>
+        ) : (
+          ''
```

Comment:
Same as in my previous comment the filter could be like:
```js
videos
  .filter({ id: { videoId } = {}} => !!videoId)
  .map(({ id: { videoId } }) => { // no need of defaults here since this was already filtered out at previous line
    //your code here
  })
```